src/components/Form/RequestForm/subcomponents/shared/FormikControl.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Input from '../Input';
+import TextArea from '../TextArea';
+import Select from '../Select';
+import Radios from '../Radios';
+import Checkboxes from '../Checkboxes';
+
+const FormikControl = (props) => {
+    const { control, ...rest } = props;
+    switch (control) {
+        case 'input':
+            return <Input {...rest} />
+        case 'textarea':
+            return <TextArea {...rest} />
+        case 'select':
+            return <Select {...rest} />
+        case 'radio':
+            return <Radios {...rest} />
+        case 'checkbox':
+            return <Checkboxes {...rest} />
+
+        default:
+            return null
+    }
+
+};
```

Comment:
this is a bit odd to me. I mention a pattern in #16 about exporting related components in one file and then importing those when consuming them… so this could look something like 

```
import Input from '../Input';
import TextArea from '../TextArea';
import Select from '../Select';
import Radios from '../Radios';
import Checkboxes from '../Checkboxes';

export { Input, TextArea, Select, Radios, Checkboxes }
```

Reply:
this is great!
thanks for the suggestion Zack